meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -21,7 +22,7 @@ mkdir -p /data_disk/docker
 mkdir -p /data_disk/resin-data
 
 # Start docker with the created image.
-docker daemon -g /data_disk/docker -s btrfs &
+docker daemon -g /data_disk/docker &
```

Comment:
aufs is just overlaying on top of underneath filesystems.
@petrosagg do you reckon we should use aufs when creating this image?